build.zig.zon

```diff
@@ -1,6 +1,8 @@
 .{
-    .name = "ansi-term",
+    .name = .ansi_term,
```

Comment:
I kinda don't want to change the package name unless it's necessary or the conventions say they should use underscores instead of dashes. Do you know of any naming conventions or examples of packages that use underscores instead of dashes?

```suggestion
    .name = .@"ansi-term",
```